hw/dv/data/sim.mk

```diff
@@ -26,9 +26,15 @@ gen_sv_flist: pre_compile prep_tool_srcs
 	@echo "[make]: gen_sv_flist"
 	cd ${build_dir} && ${sv_flist_gen_cmd} ${sv_flist_gen_opts}
 
+compile_prep := true
+ifeq (${build_cmd},vlog)
+  compile_prep := vlib work
```

Comment:
The changes to this file can perhaps be removed - see my comment on `riviera.hjson`.